corehq/apps/translations/app_translations/README.md

```diff
@@ -0,0 +1,21 @@
+Bulk Application Translations
+-----------------------------
+
+HQ supports a file download and re-upload to update all application-specific translations.
+
+The download has two variants, a multi-sheet and a single-sheet format. Both are tolerant of partial uploads.
+
+The default multi-sheet format contains a first "menus and forms" sheet for editing module and form names and menu media. It then contains a sheet for each module and each form.
```

Comment:
should the modules in this line be called menus?

Reply:
Eh, the code primarily uses "module" so I think it's fine if the docs do.